Update Airship Quests

Asheraf opened the pull request:
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed
update airship quests to match aegis script

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** 

[//]: # (Master? Slave?)
- [x] Master

**Issues addressed:**
fixes #1583 

[//]: # (Issue Tracker Number if any.)

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/herculesws/hercules/1614)
<!-- Reviewable:end -->

iRagno commented:
I think this needs also to add a function to change values for updated vars. Think in those servers who already has those scripts in use, this change will make players who already completed this quest have them incompleted and will make they need to do it again (also could cause conflicts with setquest, changequest or completequest commands, because some quest entryes may be already set if do all the quest again).

What do you think about including change on `F_ClearGarbage` function on `npc\other\Global_Function.txt` (same for juperos quest previous change) to update those vars (also including callfunc on updated npcs).

Asheraf commented:
Personally i dislike the idea of adding a function to check for the variables on every npc in the file, for servers that already use those scripts and players may have already did the quest they can run a simple sql query to change them ex:
```UPDATE `char_reg_num_db` SET `key`= 'kain_ticket' WHERE `key`= 'ein_tre'```
but this is just my opinion.

Jedzkie commented:
This effect may affected PRE-RE airship quest, maybe you create a separate files for pre-re and re?

Smokexyz commented on npc/quests/quests_airship.txt:
```diff
@@ -39,57 +40,79 @@
 		next;
 		mes "[Kain Himere]";
-		mes "A few days ago, a customer from Einbroch left this on the Airship. He contacted us later to let us know that he can't come back to the Airship to get it.";
+		mes "A few days ago, a customer";
```
`mes` with paranthesis? `mes()`
possibly in a new code-style change commit.

Smokexyz commented on npc/quests/quests_airship.txt:
```diff
@@ -2985,10 +3401,10 @@ L_Ask:
-	if (lhz_heart == 9)
+	if (yusa_01 == 9)
 	{
 		mes "["+strcharinfo(PC_NAME)+"]";
```
potential for the use of `mesf` here.

iRagno commented:
I know it is a easy task to do with a sql query, but, only admins who diffs change by change and notice that change and it's implication may take action on it.

Think on users who update without check changes trusting in emulator, they will receive bug reports from people who suddenty gets quest broken, and that "bug" will be taken as an emulator's failure.

Changing vars without a previous conversion can have effect on how players experience the game and that is the cause I propose to use that conversion.

Maybe when script was introduced devs from that time doesn't get access to official codes, and when finally have it has been notice officialy uses a different var. It is a need to make conversion, since that is the official, but conversion also need to convert previous setted vars, and that is the motive to exist to functions like clean garbage.

Asheraf commented:
Well that true, gonna apply changes after merge of pr 1548

Emistry commented:
> easy task to do with a sql query

Perhaps could add to the sql_updates table? since that will inform the admin if he hadn't execute the required SQL updates ?

Ridley8819 commented:
I agree with Emistry here.

Asheraf commented:
Updated, also removed the variable naming change for now. for pre-re i have checked the aegis script for pre-re and its the same except for the exp part which has a check in the script.
@Ridley8819 @iRagno @Jedzkie

Jedzkie commented on npc/quests/quests_airship.txt:
```diff
@@ -1593,1526 +2016,1562 @@ airplane_01,227,67,0	script	boom10#airplane	HIDDEN_WARP_NPC,{ end; }
+		mes("[Hallen]");
+		mes("So the Rune Stone");
+		mes("is an imitation of just");
+		mes("a piece of Ymir
```
Error on this line.

Jedzkie replied:
Also on this line.

Jedzkie replied:
Also this line.
